package.json

```diff
@@ -81,13 +82,6 @@
     "type": "git",
     "url": "https://github.com/aws-amplify/amplify-cli.git"
   },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "yarn build-tests-changed && yarn split-e2e-tests",
-      "pre-commit": "yarn prettier-changes && yarn lint-fix && ./scripts/package-json-check.sh && yarn verify-commit"
```

Comment:
We don't use `prettier-changes` because:
1. we need to change the staged files, not all changed files
2. `prettier-changes` doesn't stage after changing it